saleor/graphql/payment/mutations/transaction/transaction_event_report.py

```diff
@@ -57,6 +61,59 @@
     from .....plugins.manager import PluginsManager
 
 
+class CardPaymentMethodDetailsInput(BaseInputObjectType):
+    name = graphene.String(
+        description="Name of the payment method used for the transaction. Max length is 256 characters.",
+        required=True,
+    )
+    brand = graphene.String(
+        description="Brand of the payment method used for the transaction. Max length is 40 characters.",
+        required=False,
+    )
+    first_digits = graphene.String(
+        description="First digits of the card used for the transaction. Max length is 4 characters.",
+        required=False,
+    )
+    last_digits = graphene.String(
+        description="Last digits of the card used for the transaction. Max length is 4 characters.",
+        required=False,
+    )
+    exp_month = graphene.Int(
+        description="Expiration month of the card used for the transaction. Must be between 1 and 12.",
```

Comment:
```suggestion
        description="Expiration month of the card used for the transaction. Value must be between 1 and 12.",
```
maybe sounds slightly better?

Reply:
And if you agree, then the next description can be changed as well.

Reply:
Done